lib/base.ex

```diff
@@ -1,12 +1,27 @@
 defmodule FileStorageApi.Base do
   @moduledoc false
-  def api_module(module) when module in [Container, File] do
```

Comment:
I really don't like this change that you removed this it makes everything so much more complex in amount of code.
While it's not even really needed to support multiple connections only thing this needed was an extra argument.